techmatters/flex-plugins
# Referrable Resources E2E Test

## Description
This PR adds an E2E test for Referrable Resources functionality

### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-1951

### Verification steps
On `e2e-tests`, run `npm test` and check the Referrable Resources E2E test runs successfully
